js/battledata.js

```diff
@@ -811,17 +811,17 @@ var Tools = {
 		return type;
 	},
 
-	loadedSpriteData: 'xy',
+	loadedSpriteData: {'xy':1, 'bw':0},
 	loadSpriteData: function(gen) {
-		if (this.loadedSpriteData === gen) return;
-		this.loadedSpriteData = gen;
+		if (this.loadedSpriteData[gen]) return;
+		this.loadedSpriteData[gen] = 1;
 
 		var path = $('script[src*="pokedex-mini.js"]').attr('src');
 		var qs = path.split('?')[1] || '';
```

Comment:
We might want to add `'?'` here so we don't have to do the convoluted `(qs ? '?' + qs : '')` thing later.